e2e/fixtures/assets/fix_help

```diff
@@ -0,0 +1,18 @@
+Usage:
+  kics fix [flags]
```

Comment:
I'd suggest to change this command to **remediate**, WDYT?
By changing this, everywhere "fix" appears should be revised.

Reply:
For me, it is a good suggestion. I will change it.